options.go

```diff
@@ -53,6 +53,30 @@ func WithEndpointSettingsModifier(modifier func(settings map[string]*network.End
 	}
 }
 
+// WithEnv sets the environment variables for a container.
+// If the environment variable already exists, it will be overridden.
+// panics if the number of arguments is odd.
+func WithEnv(envs ...string) CustomizeRequestOption {
```

Comment:
Can you document this functional option in the `docs/features/common_functional_options.md` file? We have all the functional options that apply to modules documented there, landing on each module's page: https://golang.testcontainers.org/modules/clickhouse/#withlogconsumers